locales/en.yml

```diff
@@ -70,3 +70,22 @@ en:
       description: Discover the story of our Logo
       page_title: Logo
     title: Logo of Ruby Nord
+  talks:
+    description: |
+      Find all the past talks from our Apéros Ruby.
+      Whether you're a beginner or a confirmed developer, feel free to give a talk and share your experience with the community.
+    level:
+      all: all levels
```

Comment:
Actually we should probably do the inverse 😸 : use `all` instead of `beginner`.

I think that, as a confirmed user, you are more likely to click on slides tagged `all levels` than `beginner`.